test/check-machines-create

```diff
@@ -275,9 +275,12 @@ class TestMachinesCreate(VirtualMachinesCase):
         # We don't want to use start_vm == False because if we get a separate install phase
         # virt-install will overwrite our changes.
 
-        # Wait for virt-install to define the VM and then stop it
-        wait(lambda: "pxe-guest" in self.machine.execute("virsh list --persistent"), delay=3)
+        # Wait for virt-install to define the VM, stop it, and wait for virt-install to be done
+        wait(lambda: "pxe-guest" in self.machine.execute("virsh list --name --persistent"), delay=3)
+        wait(lambda: "pxe-guest" in self.machine.execute("virsh list --name"), delay=3)
         self.machine.execute("virsh destroy pxe-guest")
+        self.machine.execute("while pgrep virt-install; do sleep 1; done")
```

Comment:
Is this correct. Here we are waiting until virt-install to finish and process to cease existing, right?
But according to manpages, pgrep return 0 when process exists, and 1 when it doesn't exist, so the while condition should be negated, no?
```
EXIT STATUS
       0      One or more processes matched the criteria. For pkill and pidwait, one or more processes must also have been successfully signalled or waited for.
       1      No processes matched or none of them could be signalled.
```

Reply:
Right, the point of that is to wait while virt-install is running -- i.e. wait until it is *not* running any more. So this works as intended.

Reply:
No, it's already running and it works until it stops running. Precisely, while it is running (pgrep finds a process) it sleeps.